sdk/python/v1beta1/kubeflow/katib/api/katib_client.py

```diff
@@ -1583,3 +1586,112 @@ def get_trial_metrics(
             )
 
         return observation_logs.observation_log.metric_logs
+
+    def get_job_logs(
+        self,
+        name: str,  # Experiment Name
+        trial_name: str,  # Trial Name
+        follow: Optional[bool] = False,
+        namespace: Optional[str] = None,
+        container: Optional[str] = None,
```

Comment:
You can find the container name in `trial.spec.primaryContainerName`.
However you need to make sure that if:
1. Metrics Collector = StdOut, the logs are available at sidecar container with the name: `metrics-logger-and-collector`
2. Metrics collector = Push, the logs are available at the main container with the name from `trial.spec.primaryContainerName`